deeprank/generate/DataGenerator.py

```diff
@@ -33,32 +35,25 @@ def tqdm(x):
 def _printif(string, cond): return print(string) if cond else None
 
 
+_log = logging.getLogger(__name__)
+
+
 class DataGenerator(object):
 
-    def __init__(self, chain1, chain2,
-                 pdb_select=None, pdb_source=None,
-                 pdb_native=None, pssm_source=None, align=None,
+    def __init__(self, mutants, align=None,
                  compute_targets=None, compute_features=None,
                  data_augmentation=None, hdf5='database.h5',
                  mpi_comm=None):
         """Generate the data (features/targets/maps) required for deeprank.
 
         Args:
-            chain1 (str): First chain ID
-            chain2 (str): Second chain ID
-            pdb_select (list(str), optional): List of individual conformation for mapping
-            pdb_source (list(str), optional): List of folders where to find the pdbs for mapping
-            pdb_native (list(str), optional): List of folders where to find the native comformations,
-                nust set it if having targets to compute in parameter "compute_targets".
-            pssm_source (list(str), optional): List of folders where to find the PSSM files
-            align (dict, optional): Dicitionary to align the compexes,
-                                    e.g. align = {"selection":{"chainID":["A","B"]},"axis":"z"}}
```

Comment:
May need to edit and add what align does for mutants

Reply:
Actually, aligning is a bit of a problem, since it requires a plane or axis defined in 3D space. Mutants don't really represent those, they are more like a point in space, without any direction.

My suggestions: remove the alignment option, OR align along the C-alpha, C-beta bond.
What do you think?

Reply:
I think we could remove it. 
I am not sure how alignment would work for the mutants.

Reply:
OK, I'll just remove this option..

Reply:
I think we can retain realign_pdbs function

Reply:
I just removed the alignment variable. The realign_pdbs function is still there. It's just not used.

Reply:
ok thank you Coos